README.md

```diff
@@ -1,55 +1,60 @@
-# FlowCrypt: Encrypt Gmail with PGP
+# FlowCrypt: Encrypt Gmail with OpenPGP
 
 ## Users
 
-Get [FlowCrypt](https://flowcrypt.com/) browser extension at: https://flowcrypt.com/download
+Get the [FlowCrypt](https://flowcrypt.com/) browser extension from the [FlowCrypt downloads](https://flowcrypt.com/download) page.
 
 ## Developers
 
 Mock tests: ![Mock tests](https://flowcrypt.semaphoreci.com/badges/flowcrypt-browser.svg?key=d575b4ff-e35d-4217-9152-27cea9e72c19)
```

Comment:
let's remove `Mock tests: ` label (as badge displays result for all tests), and leave just image

Reply:
\+